content/php/concepts/math-functions/terms/decoct/decoct.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
The metadata needs to be formatted correctly. There shouldn't be extra blank lines in it, and the Subjects, Tags and CatalogContent need to be bulleted lists.  There's and example of properly formatted metadata in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).